src/app/routers/schema_router.py

```diff
@@ -235,3 +235,35 @@ async def get_schema_metadata_collection(
     logger.debug(f"Schemas metadata: {schema_metadata_collection}")
 
     return schema_metadata_collection
+
+
+@router.get(
+    "/v1/survey_list",
+    response_model=list[str],
+    responses={
+        500: {
+            "model": ExceptionResponseModel,
+            "content": {"application/json": {"example": erm.erm_500_global_exception}},
+        },
+        404: {
+            "model": ExceptionResponseModel,
+            "content": {
+                "application/json": {"example": erm.erm_404_no_survey_id_exception}
+            },
+        },
+    },
+)
+async def get_list_survey_id(
```

Comment:
Just a minor thing. Could be more descriptive by renaming it to get_list_unique_survey_id or get_unique_survey_list

Reply:
@IsaiahLai111 Thanks for the review, just renamed this, cheers.